CONTRIBUTING.md

```diff
@@ -99,7 +99,7 @@ in the package.
 **Code Quality**
   * Are the [coding guidelines](http://docs.astropy.org/en/latest/development/codeguide.html)
     followed?
-  * Is the code compatible with Python 2.6, 2.7, as well as >=3.3?
+  * Is the code compatible with Python 2.7 and >=3.3?
```

Comment:
Didn't we also remove some 3.3 stuff? In other words, did we bump Python 3 minversion to beyond 3.3? Maybe @bsipocz remembers.